test/01-transport-tcp.node.js

```diff
@@ -1,27 +1,48 @@
 /* eslint-env mocha */
-
 'use strict'
 
 const expect = require('chai').expect
-const parallel = require('run-parallel')
+const parallel = require('async/parallel')
 const multiaddr = require('multiaddr')
 const Peer = require('peer-info')
 const TCP = require('libp2p-tcp')
 const pull = require('pull-stream')
 
+const utils = require('./utils')
 const Swarm = require('../src')
 
 describe('transport - tcp', function () {
   let swarmA
   let swarmB
-  let peerA = new Peer()
-  let peerB = new Peer()
-
-  before(() => {
-    peerA.multiaddr.add(multiaddr('/ip4/127.0.0.1/tcp/9888'))
-    peerB.multiaddr.add(multiaddr('/ip4/127.0.0.1/tcp/9999'))
-    swarmA = new Swarm(peerA)
-    swarmB = new Swarm(peerB)
+  let peerA
+  let peerB
+
+  before((done) => {
+    utils.createInfos(2, (err, infos) => {
```

Comment:
It seems you already went ahead and did what I proposed above. Would you like to make it a module?